### PR TITLE
[#7151] ButtonSubmit//IdeaProject: add submit sticky wrapped component and us…

### PR DESCRIPTION
**Pull request by philli-m**
…e it on IdeaProject

### CODE REVIEW

File: src/components/ButtonSubmit.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
 import React from 'react'
+import { View } from 'react-native'
 import { Button } from '@rneui/base'
 
 import { styles } from './ButtonSubmit.styles'
 
+export const ButtonSubmitSticky = (props) => {
```
Comment (khamui) [inline src/components/ButtonSubmit.js:7]: personal-pref: Should we put it into its own file? it makes it a tiny little bit easier to spot I think :)

**Comment by khamui** [comment]
cool it works! but there is a greyish area above the button. Is this intended (tested on iPhone 12)?

File: src/components/ButtonSubmit.js (hunk 1 above)
Comment (philli-m) [reply src/components/ButtonSubmit.js:7]: @khamui I kind of thought of it like a container and we use this structure elsewhere like list and form fields, but maybe I should rename to include container?

Comment (philli-m) [reply src/components/ButtonSubmit.js:7]: but I also don't really mind, let me know what you prefer and i'll change it :)

**Comment by philli-m** [comment]
> cool it works! but there is a greyish area above the button. Is this intended (tested on iPhone 12)?

no I only tested on a 14, will take a look

File: src/components/ButtonSubmit.js (hunk 1 above)
Comment (khamui) [reply src/components/ButtonSubmit.js:7]: actually I like the `..Sticky` as it makes it very clear what it is. If we never use the `ButtonSubmit` standalone, we can keep it like that and just rename the file to `ButtonSubmitSticky.js`.If we do use it standalone, I would separate it so we have the variant `ButtonSubmit` and `ButtonSubmitSticky`. Does that make sense?

**Review by fuzzylogic2000 — APPROVED** [review]
Supercool! 
I don't have a grey shadow and will merge.

**Pull request merged**